src/collections/_documentation/platforms/dotnet/nlog.md

```diff
@@ -0,0 +1,169 @@
+        // Optionally specify a separate format for breadcrumbs
+        o.BreadcrumbLayout = "${logger}: ${message}";
+
+        // Debug and higher are stored as breadcrumbs (default is Info)
+        o.MinimumBreadcrumbLevel = LogLevel.Debug;
+        // Error and higher is sent as event (default is Error)
+        o.MinimumEventLevel = LogLevel.Error;
+
+        // Send the logger name as a tag
+        o.AddTag("logger", "${logger}");
+
+        // All Sentry Options are accessible here.
+    });  
+```
+
+It's also possible to initialize the SDK through the NLog integration (as opposed to using `SentrySdk.Init`). 
+This is useful when the NLog is the only integration being used in your application. To initialize the Sentry SDK through the NLog integration, provide it with the DSN:
+
+```csharp
+LogManager.Configuration = new LoggingConfiguration();
+LogManager.Configuration
+    .AddSentry(o =>
+    {
+        // The NLog integration will initialize the SDK if DSN is set:
+        o.Dsn = new Dsn("___PUBLIC_DSN___"));
+    });  
+```
+
+> NOTE:
+The SDK only needs to be initialized once. If a `DSN` is made available to this integration, by default it **will** initialize the SDK. If you do not wish to initialize the SDK via this integration, set the `InitializeSdk` flag to **false**. Not providing a DSN or leaving it as `null` instructs the integration not to initialize the SDK and unless another integration initializes it or you call `SentrySdk.Init`, the SDK will stay disabled.
+
+The SDK can also be configured via `NLog.config` XML file:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+>
+  <extensions>
+    <add assembly="Sentry.NLog" />
+  </extensions>
+
+  <targets>
+    <target xsi:type="Sentry" name="sentry"
+            dsn="___PUBLIC_DSN___"
+            layout="${message}"
+            debug="true"
+            breadcrumbLayout="${message}"
+            minimumBreadcrumbLevel="Debug"
+            minimumEventLevel="Error">
+
+      <!-- Advanced options can be configured here-->
+      <options
+          environment="Development"
+          attachStacktrace="true"
+          sendDefaultPii="true"
+          shutdownTimeoutSeconds="5"
+        >
+        <!--Advanced options can be specified as attributes or elements-->
+        <includeEventDataOnBreadcrumbs>true</includeEventDataOnBreadcrumbs>
+      </options>
+
+      <!--Add any desired additional tags that will be sent with every message -->
+      <tag name="logger" layout="${logger}" />
+      <tag name="example" layout="sentry-nlog" />
+    </target>
+  </targets>
+
+  <rules>
+    <logger name="*" writeTo="sentry" />
+  </rules>
+</nlog>
+```
+
+### Options
+
+#### MinimumBreadcrumbLevel
+
+A `LogLevel` which indicates the minimum level a log message has to be included as a breadcrumb. By default this value is `Info`.
```

Comment:
```suggestion
A `LogLevel` which indicates the minimum level a log message needs to be, for Sentry to receive it as an event. By default, this value is `Info`.
```

Reply:
I think this might be a more appropriate change:

`LogLevel` that indicates the minimum level a log message needs to be in order for Sentry to receive it as a breadcrumb in an event. By default, this value is `Info`.

_(it should still specify that the message is being sent as a breadcrumb, not an event)_

Reply:
I hit commit suggestion without scrolling all the way. @josh-degraw is right, it's to be included as breadcrumb, not event. So the original text was right although perhaps confusing as most of what I write.